components/Support/Choices.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div
+    class="flex flex-wrap gap-2"
+    role="listbox"
+    :aria-labelledby="question.title"
+    tabindex="0"
+    :aria-activedescendant="active"
+    @keydown="handleKeyPressForActiveDescendant"
+    @keydown.space="selectActiveChoice"
+    @keydown.enter="selectActiveChoice"
+    @focusout="focusOut"
+  >
+    <template v-if="'choices' in question">
+      <SupportChoice
+        v-for="choice in question.choices"
+        :id="choice.id"
+        :key="choice.id"
+        :active="isActive(choice.id)"
+        :selected="choice.id === selected"
+        @select="$emit('select', choice.id)"
+      >
+        {{ choice.label }}
+      </SupportChoice>
+    </template>
+    <template v-else-if="'segment' in question">
+      <form
+        class="w-4xl"
+        @submit.prevent="submit(question)"
+      >
+        <InputGroup
+          v-model="form.email"
+          label="Votre adresse email"
+          type="email"
+          :required="true"
+          :has-error="!!getFirstError('email')"
+          :error-text="getFirstError('email')"
+          @blur="touch('email')"
+        />
+        <InputGroup
+          v-model="form.subject"
+          label="Le sujet de votre question"
+          type="text"
+          :required="true"
+          :has-error="!!getFirstError('subject')"
+          :error-text="getFirstError('subject')"
+          @blur="touch('subject')"
+        />
+        <InputGroup
+          v-model="form.body"
+          label="Votre question"
+          type="textarea"
+          :required="true"
+          :has-error="!!getFirstError('body')"
+          :error-text="getFirstError('body')"
+          @blur="touch('body')"
+        />
+        <div
+          v-if="messageSent"
+          class="flex w-56 text-center"
+        >
+          <SimpleBanner type="primary">
+            Message envoyé !
```

Comment:
This is not translated?

Reply:
No the support page is only in french, with the corresponding `lang` attribute because the answers and configuration are only in french